demo/molecule/fieldTextarea/playground

```diff
@@ -0,0 +1,62 @@
+const styleList = {
```

Comment:
you could add one example with less `maxCharacters` in order to test it easier

Reply:
👍 `maxChars={30}`